wbs/1.02C.09.tex

```diff
@@ -0,0 +1,16 @@
+\subsection{1.02C.09 Integration and Test}
+
+This WBS element covers integration and test activities. It includes:
+
+\begin{itemize}
+
+\item{Test and QA Data Curation: Curation of fixed data sets and
+  recommended tests assembled to provide a small but rich set of test
+  data for 10.2.1.01}
+  
+\item{Development of testing specifications and operation rehearsals
```

Comment:
I assume indeed NCSA do a lot of the integration 02C.07.1 as Margaret says in her mail. 
So I agree the suggested title indeed would be more appropriate